include/fem-dem/vans_assemblers.h

```diff
@@ -497,6 +497,39 @@ class GLSVansAssemblerMagnus : public ParticleFluidAssemblerBase<dim>
     lagrangian_physical_properties;
 };
 
+/**
+ * @brief Class that assembles the one way coupled Viscous Torque dissipation as defined by Lightfoot, Bird & Stewart (2002) - Transport Phenomena, 2nd Ed. - Example 3.6.5
+ * M_viscous = 8 * MPI * mu * dp^3 * omega
```

Comment:
I am not a fan of this. This version is not useful, because it does not work if the fluid has a vorticity. I would at least put the version from Derksen et al. with the fluid vorticity in. The present one is kinda useless and dangerous to have...